settings.py

```diff
@@ -141,4 +141,4 @@
 TEMPLATE_DIR = os.path.join(CERT_PRIVATE_DIR, TEMPLATE_DATA_SUBDIR)
 
 with open(os.path.join(CERT_PRIVATE_DIR, CERT_DATA_FILE)) as f:
-    CERT_DATA = yaml.load(f.read())
+    CERT_DATA = yaml.load(f.read().decode("utf-8-sig"))
```

Comment:
Look https://docs.python.org/2/library/codecs.html#encodings-and-unicode .
I was actually trying to prevent issues with a cert-data.yml saved with Notepad on Windows that add BOM chars saving in UTF8.  (will not ever my case !!!!)
BTW if you agree I can change it in just utf-8